### DISCUSSION TITLE
进阶路线图 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#169] [Announcements]
# 进阶路线图 - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%9C%BA%E5%99%A8%E5%AD%A6%E4%B9%A0%E8%BF%9B%E9%98%B6/roadmap/

**Comment by vx1day** [comment]
长夜漫漫，无心睡眠，翻开指南，开始拜读。🐱‍🏍

**Comment by Karry2021** [comment]
thanks for your sharing.

**Comment by EGalahad** [comment]
建议用AoS来预习概率与统计课程吗？

**Reply by PKUFlyingPig** [reply]
推荐一本我个人很喜欢的书 https://www.probabilitycourse.com/

**Comment by ZhitongGuo** [comment]
请问作者一般用什么工具自学书/论文呢？感谢🙏

**Reply by PKUFlyingPig** [reply]
电子书我一般用neatreader，跨平台同步做的比较好。读论文就普通pdf阅读器。

**Comment by crystal-cheung** [comment]
感谢分享

**Reply by crystal-cheung** [reply]
还想放上一个useful的知乎回答 CS294 深度增强学习 这门课的质量是不是不大好？ - Buoyancy的回答 - 知乎
https://www.zhihu.com/question/61171437/answer/876455948 类似于伯克利的学习路径

**Reply by zhusq20** [reply]
补充一下，CS294是CS285的前身

**Comment by lwtwl23** [comment]
Nit: Parametric Inference是AOS的第9章，不是第8章

**Comment by Murasamess** [comment]
有个问题哈，PRML8章节之前的内容，应该怎么跟AOS进行结合呢？还是说先看完PRML前七章，再开始与AOS结合？

**Comment by JaredforReal** [comment]
大佬大佬有没有robotics相关的课程推荐呀

**Comment by cgwer** [comment]
看不懂
